File: loops/fizzbuzz.c
//zoey sosa, FizzBuzz C
#include <stdio.h>

int main(){
    for (int x = 1; x <= 50; x++){
        if (x % 3 == 0 && x % 5 == 0) {
            printf("fizzbuzz\n");
        } else if (x % 3 == 0) {
            printf("fizz\n");
        } else if (x % 5 == 0) {
            printf("buzz\n");
        } else { 
            printf("%d\n", x);
        }
    }
    return 0;
}

File: loops/fizzBuzz.py
# Zoey Sosa, FizzBuzz python

for x in range(0,51):
    if x % 3 == 0 and x % 5 == 0:
        print("fizzbuzz")
    elif x % 3 == 0:
        print("Fizz")
    elif x % 5 == 0:
        print("buzz")
    else:
        print(x)